tests/errors.spec.js

```diff
@@ -0,0 +1,34 @@
+var m = require('mochainon')
```

Comment:
Some tests should actually be in JS as TS won't let you go some wrong paths. But this one can be TS indeed.